src/common/http.hpp

```diff
@@ -40,13 +40,15 @@
 #include <common/log.hpp>
 
 namespace mender {
-namespace update {
-namespace http_resumer {
+namespace common {
+namespace http {
+namespace resumer {
 class DownloadResumerClient;
 class HeaderHandlerFunctor;
 class BodyHandlerFunctor;
```

Comment:
These can be moved few lines below to inside of `mender::common::http` as I tried to show you on my previous comment [here](https://github.com/mendersoftware/mender/pull/1598#discussion_r1514005108).